e-shop-manager/config/java.header

```diff
@@ -0,0 +1,4 @@
+^/{68}$
+^/{2}[ ]{1}[A-Z]{1}[a-z]+[ ]{1}[A-Z]{1}[a-z]+[ ]{1}\d{7}$
```

Comment:
Ma l'header non bastava mettessi i nostri nomi e le matricole, dici che sia da mettere qualsiasi nome?

Reply:
Si in effetti è sufficiente mettere i nostri nomi